packages/webpack/src/buildtime/exclude.js

```diff
@@ -2,7 +2,7 @@
 /** @typedef {import('./policyGenerator').InspectableWebpackModule} InspectableWebpackModule */
 
 const path = require('node:path')
-const EXCLUDE_LOADER = path.join(__dirname, '../excludeLoader.js')
+const EXCLUDE_LOADER = path.resolve(path.join(__dirname, '../excludeLoader.js'))
```

Comment:
```suggestion
const EXCLUDE_LOADER = path.join(__dirname, '../excludeLoader.js')
```

`__dirname` is always absolute, so this `path.resolve()` call is redundant.

Reply:
:shaking_face: